README.md

```diff
@@ -3,10 +3,9 @@
 This project is an open source Job Board. It's primary purpose is to serve as a
 training exercise.
 
-
 ## Getting Started
 
-1. Clone the repo `git vlone git@github.com:MattGibney/JobBoard.git`
+1. Clone the repo `git clone https://github.com/MattGibney/JobBoard.git`
```

Comment:
Good spot on the typo.

I'd rather not change the repo URL though, It's intentionally like this. Using SSH is much better than HTTPS (In my opinion).

Reply:
No problem, I'll take your lead on that.  I didn't realise that was what the difference was so good to know.